src/main/java/wolox/training/exceptions/BookNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package wolox.training.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import wolox.training.constants.ExceptionConstants;
+
+@ResponseStatus(code = HttpStatus.NOT_FOUND, reason = ExceptionConstants.BOOK_NOT_FOUND)
+public class BookNotFoundException extends RuntimeException {
+
+  public BookNotFoundException() {
+    super(ExceptionConstants.BOOK_NOT_FOUND, new Exception());
```

Comment:
I think you can either:
a) use `this` instead of a new Exception in this constructor
b) or use `super(ExceptionConstants. BOOK_NOT_FOUND);` (I believe there is that constructor too)